The code/Task2.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+
+class Task {
+  String descriptask;
+  bool isCompleted;
+
+  Task(this.descriptask, this.isCompleted);
+}
+
+class Todolist {
+  final List<Task> listTask = [];
+
+  void displayinfo() {
+    print("Task List: ");
+    for (var i = 0; i < listTask.length; i++) {
+      final task = listTask[i];
+      final iscompleted = task.isCompleted ? '[//]' : '[?]';
+      print("$i $iscompleted ${task.descriptask}");
+    }
+  }
+
+  void addTask(var description) {
+    final task = Task(description, false);
+    listTask.add(task);
+  }
+
+  void completask(var taskindex) {
+    if (taskindex >= 0 && taskindex < listTask.length) {
+      listTask[taskindex].isCompleted = true;
+    }
+  }
+
+  void ubdateTASK(var tasknumber, var newtask) {
+    if (tasknumber != null && tasknumber >= 0 && tasknumber < listTask.length) {
+      listTask[tasknumber].descriptask = newtask;
+    }
+  }
+
+  void deletetask(var deletindex) {
+    if (deletindex != null && deletindex >= 0 && deletindex < listTask.length) {
+      listTask.removeAt(deletindex);
+    }
+  }
+}
+
+void main() {
+  final ToDolist = Todolist();
+  while (true) {
+    ToDolist.displayinfo();
+    print("Select One Of These Options: ");
+    print("1: ADD TASK");
+    print("2: DELET TASK");
```

Comment:
I recommend downloading a spell checker extension on your vsCode. It will ensure you use proper programming variable naming conventions and help you avoid making a misspelling